test/e2e/app-dir/index.test.ts

```diff
@@ -52,11 +52,16 @@ describe('views dir', () => {
     expect(html).toContain('hello world')
   })
 
-  it('should serve from root', async () => {
+  it('should serve from app', async () => {
     const html = await renderViaHTTP(next.url, '/dashboard')
     expect(html).toContain('hello from root/dashboard')
   })
 
+  it('should serve /index as separate page', async () => {
+    const html = await renderViaHTTP(next.url, '/dashboard/index')
```

Comment:
Are we not resolving without the `/index` being in the URL correctly? Seems like this should 404 and only be accessible at `/dashboard`

Reply:
This was changed in the RFC. You'll be able to colocate `/dashboard/page.js` and `/dashboard/layout.js`. `/dashboard/index/page.js` would resolve to `/dashboard/index`. Basically removing `index` being special.